client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx

```diff
@@ -29,12 +30,12 @@ export const ChapterPage: NextPage = () => {
     <Layout>
       <Card className={styles.card}>
         <ProgressCardContent loading={loading}>
-          <Heading as="h5" fontWeight="normal">
+          <Heading fontSize={'md'} as="h1" fontWeight="semibold">
             {data.chapter.name}
           </Heading>
           <Box>
             <Link href={`${chapterId}/users`} target="_blank">
-              Chapter Users
+              Chapter User
```

Comment:
```suggestion
              Chapter Users
```
Link leads to the list of chapter users.